lang/src/org/partiql/lang/types/ScalarTypeExtensions.kt

```diff
@@ -1,33 +0,0 @@
-package org.partiql.lang.types
-
-// TODO: remove this file once we use OTS to define aliases & arity for a scalar type
-
-enum class BuiltInScalarType(
-    val typeAliases: List<String>,
-    val arity: IntRange
-) {
-    BLOB(listOf("blob"), 0..0),
-    BOOLEAN(listOf("bool", "boolean"), 0..0),
-    CHARACTER(listOf("char", "character"), 0..1),
-    CHARACTER_VARYING(listOf("varchar", "character_varying"), 0..1),
-    CLOB(listOf("clob"), 0..0),
-    DATE(listOf("date"), 0..0),
-    DECIMAL(listOf("dec", "decimal"), 0..2),
-    DOUBLE_PRECISION(listOf("double_precision"), 0..0),
-    FLOAT(listOf("float"), 0..1),
-    INTEGER4(listOf("int4", "integer4"), 0..0),
-    INTEGER8(listOf("int8", "bigint", "integer8"), 0..0),
-    INTEGER(listOf("int", "integer"), 0..0),
-    NUMERIC(listOf("numeric"), 0..2),
-    REAL(listOf("real"), 0..0),
-    SMALLINT(listOf("int2", "smallint", "integer2"), 0..0),
-    STRING(listOf("string"), 0..0),
-    SYMBOL(listOf("symbol"), 0..0),
-    TIME(listOf("time"), 0..1),
-    TIME_WITH_TIME_ZONE(listOf("time_with_time_zone"), 0..1),
-    TIMESTAMP(listOf("timestamp"), 0..0);
-}
-
-@JvmField internal val TYPE_ALIAS_TO_SCALAR_TYPE = BuiltInScalarType.values().flatMap { scalarType ->
-    scalarType.typeAliases.map { typeAlias -> typeAlias to scalarType }
-}.associate { it.first to it.second }
```

Comment:
This file is introduced in https://github.com/partiql/partiql-lang-kotlin/pull/753/files#diff-c3e6084bad083065b5a40e9feb4a8d84bcdf86289ae0d42c3628eb09c7339ac9 temporarily, to summarize all the type aliases and type arity defined here in `main`. branch, https://github.com/partiql/partiql-lang-kotlin/blob/a0209df1b22f831a754eb11b070d0b7f9231c78a/lang/src/org/partiql/lang/syntax/LexerConstants.kt.

Reply:
Since now we are exposing interfaces to allow users to define type aliases & type validation for a certain scalar type, we are removing it.